lib/index.js

```diff
@@ -40,6 +40,13 @@ module.exports = function (gulp, options) {
     }
   });
 
+  if (!options.allowUnhandledRejections) {
+    process.on('unhandledRejection', function(e) {
```

Comment:
Since promises swallow by default, might want to add a *why was this thrown* to this.